[DOCS-1921] Release notes for v0.76.1 (#1935)
wandb/docs: release-notes/server-releases.mdx

<EolReminder/>

> <Update label="v0.76.1" description="December 3, 2025">

Author: ```suggestion
<Update label="v0.76.1" description="December 3, 2025">
**December 3, 2025**
```

## Fixes

> - Fixed a bug when importing a line plot from a workspace to a report could get stuck if extreme outliers were hidden in the plot.

Author: ```suggestion
- Fixed a bug where importing a line plot from a workspace to a report could get stuck if extreme outliers were hidden in the plot.
```

- Fixed a bug that could prevent a table panel from resizing correctly if its height was reduced.

> - Fixed a minor display bug where bounding box overlays be slightly misaligned.

Author: ```suggestion
- Fixed a bug where mask overlays could overflow an image with a caption.
```

<EolReminder/>

<Update label="v0.76.1" description="December 3, 2025">

> **December 3, 2025**

Author: ```suggestion
**December 3, 2025**

```

<EolReminder/>

<Update label="v0.76.1" description="December 3, 2025">
**December 3, 2025**

## Fixes
- Fixed a bug where importing a line plot from a workspace to a report could get stuck if extreme outliers were hidden in the plot.
- Fixed a bug that could prevent a table panel from resizing correctly if its height was reduced.

> - Fixed a bug where mask overlays could overflow an image with a caption.

Author: ```suggestion
- Fixed a bug where mask overlays could overflow an image with a caption.
- When a client cancels a GraphQL operation, it is now correctly reported as a `4xx` HTTP client error rather than a `5xx` HTTP server error. 
```

<EolReminder/>

<Update label="v0.76.1" description="December 3, 2025">
**December 3, 2025**

## Fixes
- Fixed a bug where importing a line plot from a workspace to a report could get stuck if extreme outliers were hidden in the plot.
- Fixed a bug that could prevent a table panel from resizing correctly if its height was reduced.
- Fixed a bug where mask overlays could overflow an image with a caption.

> - When a client cancels a GraphQL operation, it is now correctly reported as a `4xx` HTTP client error rather than a `5xx` HTTP server error. 

Author: ```suggestion
- Fixed a bug where clicking the **Service Accounts** tab in the Organization Dashboard could cause the page to crash.
- Fixed a caching bug where the wrong run's logs could display when viewing run logs and switching from one run to another.
- When a client cancels a GraphQL operation, it is now correctly reported as a `4xx` HTTP client error rather than a `5xx` HTTP server error. 
```

<EolReminder/>

<Update label="v0.76.1" description="December 3, 2025">
**December 3, 2025**

## Fixes
- Fixed a bug where importing a line plot from a workspace to a report could get stuck if extreme outliers were hidden in the plot.
- Fixed a bug that could prevent a table panel from resizing correctly if its height was reduced.
- Fixed a bug where mask overlays could overflow an image with a caption.

> - Fixed a bug where clicking the **Service Accounts** tab in the Organization Dashboard could cause the page to crash.

Author: ```suggestion
- Fixed a bug where a run note's edit box's size was too narrow for comfortable editing. Now the edit box uses the same dimensions as the text in display mode.
- Fixed a bug where clicking the **Service Accounts** tab in the Organization Dashboard could cause the page to crash.
```

<EolReminder/>

> <Update label="v0.76.1" description="December 3, 2025">

Author: ```suggestion
<Update label="v0.76.1" description="December 4, 2025">
```
